.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,58 @@
+name: Deploy to Netlify
+
+on:
+  push:
+    branches:
+      - main
+  repository_dispatch:
+    types: [zui-docs-update]
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy to Netlify
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build site
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: yarn
+      - name: Build
+        run: make build
+      # Popular action to deploy to Netlify:
+      # Docs: https://github.com/marketplace/actions/netlify-actions
+      - name: Deploy to Netlify
+        uses: nwtgck/actions-netlify@v1.2
+        with:
+          # Build output to publish to Netlify
+          publish-dir: ./build
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          production-branch: init
+          production-deploy: true
+        env:
+          NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
+          NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
+      - name: Check for broken links
+        uses: ruzickap/action-my-broken-link-checker@v2
+        with:
+          url: https://zui.brimdata.io
+          cmd_params:
+            --verbose
+            --buffer-size=8192
+            --max-response-body-size 1000000000
+            --header="User-Agent:Mozilla/5.0"
+            --header="Accept-Encoding:zstd,br,gzip,deflate"
+            --exclude algolia.net
+            --exclude www.googletagmanager.com
+            --exclude https://support.virustotal.com/hc/en-us/articles/115002146769-Comments
+            --exclude https://github.com/brimdata/brimcap#
+            --exclude https://github.com/brimdata/build-suricata.*#
+            --exclude https://github.com/brimdata/brimcap/wiki.*#
```

Comment:
The link checker has some extra stuff compared to our link checker configs elsewhere, due to some fussy content we link to from the Zui docs.

1. The `User-Agent` config was necessary because the link to the AWS Security Group page failed with an HTTP 403 unless hit with a browser-like User Agent
2. The exclusions to the GitHub URLs containing `#` anchors is due to a known limitation of Muffet (https://github.com/raviqqe/muffet/issues/144)
3. The VirusTotal exclusion is because their page we link to seems to return an HTTP 403 even with the User Agent set
4. The large `max-response-body-size` is so link checks to bulky video content will succeed